### DISCUSSION TITLE
Annotation Feature Unavailable

### DISCUSSION THREAD

**Discussion by avharris** [sumatrapdfreader/sumatrapdf#3058] [General]
I have v 3.4.6 and there is no annotation feature on the menu under selection. I could not find any information elsewhere. What am I missing?
![SelectionMenu](https://user-images.githubusercontent.com/15877204/198300059-568c71ff-f643-42b5-a387-f63f4215ff07.png)

**Comment by GitHubRulesOK** [comment]
the selection needs context so context key or `a` command or use ctrl K menu or right click
![image](https://user-images.githubusercontent.com/12754913/198329889-a02a614e-350d-4c17-ae52-78d8854ba321.png)

**Comment by avharris** [comment]
Here is what I get whatt I select a word and right-click.
![SelectionMenu2](https://user-images.githubusercontent.com/15877204/198356941-00ad6a4e-75d0-4d84-ad54-27558fdf52f7.png)

**Reply by GitHubRulesOK** [reply]
double check it is a PDF in 3.4.6
![image](https://user-images.githubusercontent.com/12754913/198385679-ace78a2f-de3f-47eb-bd3b-16899018eeac.png)
![image](https://user-images.githubusercontent.com/12754913/198386389-8a31e06c-c620-42f8-860c-9cea0e5f6402.png)

**Comment by avharris** [comment]
Thank you. The program is so good at reading different formats that I didn't realize that this was not a PDF. I can annotate for a PDF.

Too bad it doesn't work on other file types.

**Reply by GitHubRulesOK** [reply]
Hmm it used to allow highlight text, etc. but as experimental and specific to multi-format, it had to eventually be dropped for concentrating on Mu.PDF compatibility.
3.2 viewer can still read hand crafted ePub/text/chm fixed ui highlights, but not easy to build.smx without the earlier experimental versions see https://github.com/sumatrapdfreader/sumatrapdf/issues/2970#issuecomment-1253052409

![image](https://user-images.githubusercontent.com/12754913/198398732-1298b23c-be96-4b4e-91cf-47dc084a57ca.png)
